linalg_ops/convolution/generated/f16_f16_f16/conv2d_f16_f16_f16_large.mlir

```diff
@@ -0,0 +1,8 @@
+func.func @conv2d_accumulate_2_4_128_128_times_3_3_8_dtype_f16_f16_f16(%lhs: tensor<2x4x128x128xf16>, %rhs: tensor<8x4x3x3xf16>, %acc: tensor<2x8x126x126xf16>) -> tensor<2x8x126x126xf16> {
```

Comment:
Do we need to add these generated test cases to the repo?

Reply:
That was an intentional design decision for the test suite repo. See this section of https://github.com/iree-org/iree-test-suites/pull/22:

> ### Ran the `generate_e2e_matmul_tests.py` script offline and checked in the generated files
> * Currently 56 files totaling 1.90MB on disk (~27000 lines of code according to GitHub)
> * Now we can inspect the test cases without needing to run the generator locally, and I fixed a few formatting issues
> * I think this makes test suite management easier, and having the generated files in this test suites repository doesn't cost the main repository much (just extra `git checkout` time), but I could see a case for more tightly coupling the generator with the test runner